apps/src/templates/CensusForm.jsx

```diff
@@ -41,15 +42,36 @@ const styles = {
 class CensusForm extends Component {
 
   state = {
-    value: ''
+    submission: {
+      name: '',
+      email: '',
+      role: '',
+      pledge: false,
+    }
   };
 
-  handleChange = event => {
-    this.setState({value: event.target.value});
+  handleChange = (propertyName, event) => {
```

Comment:
Because you end up binding calls to handleChange anyways (so that you get your propertyName), there's no need to use this form, and you can instead do `handleChange(propertyName, event) {`